### PR TITLE
283c: The Science of Insecurity

### PR DESCRIPTION
**Pull request by AmosAidoo**

### CODE REVIEW

**Comment by hellerve** [comment]
Thanks for your contribution!

Just a heads-up: I’m currently on vacation, travelling around, and will probably not be able to watch the talk/review the PR until early next week.

**Comment by AmosAidoo** [comment]
> Thanks for your contribution!
> 
> Just a heads-up: I’m currently on vacation, travelling around, and will probably not be able to watch the talk/review the PR until early next week.

Sure no problem🙂. You are definitely going to enjoy the talk

File: README.md [hunk 1]
```diff
@@ -1120,6 +1120,7 @@ ever.
 * [Boot strapping slightly more secure systems](https://media.ccc.de/v/33c3-8314-bootstraping_a_slightly_more_secure_laptop/oembed) [00:47:43] by **Trammell Hudson** (2016)
 * [Repsych: Psychological Warfare in Reverse Engineering](https://www.youtube.com/watch?v=HlUe0TUHOIc) [00:38:48] by **Chris Domas** (2015)
 * [Pulling Back the Curtain on Airport Security: Can a Weapon Get Past TSA](https://www.youtube.com/watch?v=hbqVNlwfjxo) [00:31:52] by **Billy Rios** (2014)
+* [283c: The Science of Insecurity](https://www.youtube.com/watch?v=3kEfedtQVOY) [00:59:22] by **Meredith L. Patterson, Sergey** (2011)
```
Comment (hellerve) [inline README.md]: Looks like there is a typo here:
```suggestion
* [28c3: The Science of Insecurity](https://www.youtube.com/watch?v=3kEfedtQVOY) [00:59:22] by **Meredith L. Patterson, Sergey** (2011)
```

**Review by hellerve — CHANGES_REQUESTED** [review]
Well that was fun!

Just one minor thing, then we’re good to go!

**Comment by AmosAidoo** [comment]
I've made the change, thanks

**Review by hellerve — APPROVED** [review]
Thank you 😸

**Pull request merged**